tests/Integration/Foundation/MaintenanceModeTest.php

```diff
@@ -24,8 +23,6 @@ protected function setUp(): void
         });
 
         parent::setUp();
-
-        $this->withoutMiddleware(TestbenchPreventRequestsDuringMaintenance::class);
```

Comment:
It should use https://github.com/laravel/framework/blob/11.x/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php